core/base/compactTriangulationPreconditioning/Octree.h

```diff
@@ -0,0 +1,432 @@
+/// \ingroup base
+/// \class Octree
```

Comment:
It seems that this octree implementation is fairly generic (it is not specific to the CompactTriangulation) and that it could be useful for others as well. 
If so, please move that class to a new directory `core/base/octree/`

Reply:
solved.